sound/soc/sof/ipc.c

```diff
@@ -1039,6 +1039,7 @@ int sof_ipc_init_msg_memory(struct snd_sof_dev *sdev)
 struct snd_sof_ipc *snd_sof_ipc_init(struct snd_sof_dev *sdev)
 {
 	struct snd_sof_ipc *ipc;
+	const struct ipc_ops *ops;
```

Comment:
I just realized we don't have an 'sof' prefix for the structure. Should we use sof_ipc_ops?

Reply:
Yes, I think it is warranted

Reply:
Hrm, then should we rename all the other ops struct, like
struct ipc_pcm_ops
struct ipc_tplg_ops
struct ipc_tplg_control_ops

@ranj063, @plbossart ?

Reply:
yes, but I wonder if we can do this easily with fixups, or it's too late already and we just accept this and move on.

Reply:
I can add the fixups to this series as I did with the sof_ipc_ops, if that's what we want. Separate PR will conflict with everything ;)

Reply:
ok, let's do a fixups series then.

Reply:
@plbossart, I have taken a look on how to do a fixup for the missing `sof_` prefixes in the existing ops, but it is not going to be nice. We can not fix up by individual commits as we will break compilation, There were also a bit of churn in the files, so the fixup is not going to apply cleanly. Let me see.

I'll drop the fixup from this PR and create a new one. They will conflict big time, so one will need to be rebased depending on the order.